### PR TITLE
update hadoop major version 2 and yarn for cdh4

### PR DESCRIPTION
**Pull request by fparisotto**

### CODE REVIEW

**Comment by luisguilherme** [comment]
LGTM

File: ephemeral-hdfs/init.sh [hunk 1]
```diff
@@ -17,22 +17,22 @@ case "$HADOOP_MAJOR_VERSION" in
     sed -i 's/-jvm server/-server/g' /root/ephemeral-hdfs/bin/hadoop
     ;;
   2) 
-    wget http://s3.amazonaws.com/spark-related-packages/hadoop-2.0.0-cdh4.2.0.tar.gz
```
Comment (luisguilherme) [inline ephemeral-hdfs/init.sh:20]: Keep this as is (older version). Change only the `yarn)` entry

File: persistent-hdfs/init.sh [hunk 2]
```diff
@@ -16,22 +16,22 @@ case "$HADOOP_MAJOR_VERSION" in
     mv hadoop-1.0.4/ persistent-hdfs/
     ;;
   2)
-    wget http://s3.amazonaws.com/spark-related-packages/hadoop-2.0.0-cdh4.2.0.tar.gz
```
Comment (luisguilherme) [inline persistent-hdfs/init.sh:19]: Keep this as is (older version). Change only the `yarn)` entry

**Comment by luisguilherme** [comment]
Minor suggestion, LGTM

File: ephemeral-hdfs/init.sh (hunk 1 above)
Comment (douglaz) [inline ephemeral-hdfs/init.sh:20]: Perhaps it makes sense to change here and keep the YARN.

Comment (luisguilherme) [inline ephemeral-hdfs/init.sh:20]: This cloudera distribution is yarn-based.

**Comment by luisguilherme** [comment]
Ah, please add spark 1.3.0 support here altogether. A simple merge from mesos/v1.3.0 should do it.

File: ephemeral-hdfs/init.sh (hunk 1 above)
Comment (douglaz) [inline ephemeral-hdfs/init.sh:20]: So I would suggest to change both, or merge them.

Comment (fparisotto) [inline ephemeral-hdfs/init.sh:20]: I think our scripts are not fully compatible with yarn setup. Maybe is better to leave yarn) with hadoop-2.4

**Pull request merged**